main.go

```diff
@@ -103,16 +110,27 @@ func clear(bin string) {
 	cmd.Run()
 	fmt.Println("$ gohr " + bin)
 }
-func init() {
-	testing.Init()
```

Comment:
[FYI]
If testing.Init() executed, `flag.PrintDefaults` includes test option.
We don't want to include them, so decide not to parse the flag in init().